doc/source/contribute.rst

```diff
@@ -1,130 +1,145 @@
+
+    + Review the conflicts (`VS Code <https://code.visualstudio.com/>`__ is a great tool for this)
+    + Mark them as resolved ``git add <filename>``
+    + Continue the rebase ``git rebase --continue``
+* Push your changes to your fork: |br|
+  ``git push origin <branchname>`` |br|
+  This step synchronizes your local branch and the branch in your fork. It is not required after every commit, but it is certainly necessary once you are ready to merge your code into ``upstream``.
 
   .. tip::
-     It can be useful to push your local branch to your fork under a different
-     name using ``git push origin <local-branchname>:<remote-branchname>``
+     It can be useful to push your local branch to your fork under a different name using: |br|
+     ``git push origin <local-branchname>:<remote-branchname>``
 
-Once you think your contribution is finished and can be merged into the main
-repository
+Once you think your contribution is finished and can be merged into the main repository
 
 * Make sure your changes are rebased and pushed into your fork
-* Log into GitHub with your account and create a pull request (merge your fork
-  branch into the main repository master branch)
-* While open, commits pushed to your fork branch will automatically update the
-  pull request
+* Log into GitHub with your account and create a pull request (merge your fork branch into the main repository master branch)
+* While open, commits pushed to your fork branch will automatically update the pull request
+
+.. _DevelopmentalSetup:
+
+Developmental set-up
+--------------------
+
+If you want to develop new functionality for pycompwa, the :ref:`PyPI setup <InstallFromPyPI>` won't suffice and you will have to :ref:`build from source <InstallFromSource>`. In addition, it is useful to use a `virtual environment wrapper <https://virtualenvwrapper.readthedocs.io/en/latest/>`__ instead of using a local ``pip`` install of pycompwa. This way, you directly call your local pycompwa source code, so you can modify and test things immediately. It is advised to work within a virtual environment for this set-up as well, just as in the normal :ref:`Installation` procedure.
+
+First, go into your pycompwa folder, then un-install the pycompwa install from PyPI (if necessary), and set up the virtual environment wrapper: |br|
+
+.. code-block:: shell
+
+  pip uninstall pycompwa # if necessary
+  pip install virtualenvwrapper
+  source venv/bin/virtualenvwrapper.sh
+  add2virtualenv .
+
+In this example, we assume you use Python's ``venv`` and that you set your virtual environment is set up in a folder called ``venv`` (see 3rd line). You can also work with other virtual environment handlers, such as `Conda <https://docs.conda.io/en/latest/>`__.
+
+Now, go through the standard build procedure:
```

Comment:
I would scratch standard

Reply:
> I made some suggestions. The part about referring to building from source in the development mode is the most important one.

I think in general we should simplify the [Installation](https://pycompwa.readthedocs.io/en/latest/installation.html) page: stick with one virtual environmant (Conda?), then specifically write how to set it up, and write (at most) in a note that you can also work with `venv` etc.

The installation page should also be more simple: only explain for instance `pip install` and then refer to the Contribute page for the 'developmental setup' (i.e. build from source).